Package.swift

```diff
@@ -1,32 +1,16 @@
-//
-//  Package.swift
-//  Kingfisher
-//
-//  Created by Wei Wang on 16/1/22.
-//
-//  Copyright (c) 2018 Wei Wang <onevcat@gmail.com>
-//
-//  Permission is hereby granted, free of charge, to any person obtaining a copy
-//  of this software and associated documentation files (the "Software"), to deal
-//  in the Software without restriction, including without limitation the rights
-//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-//  copies of the Software, and to permit persons to whom the Software is
-//  furnished to do so, subject to the following conditions:
-//
-//  The above copyright notice and this permission notice shall be included in
-//  all copies or substantial portions of the Software.
-//
-//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-//  THE SOFTWARE.
-
+// swift-tools-version:4.2
 import PackageDescription
 
 let package = Package(
-  name: "Kingfisher",
-  exclude: ["Tests", "Sources/AnimatedImageView.swift", "Sources/UIButton+Kingfisher.swift", "Sources/WKInterfaceImage+Kingfisher.swift"]
+    name: "Kingfisher",
+    // platforms: [.iOS("10.0"), .macOS("10.12"), tvOS("10.0"), .watchOS("3.0")],
```

Comment:
Is it related to [this proposal](https://github.com/apple/swift-evolution/blob/master/proposals/0236-package-manager-platform-deployment-settings.md)? Is that mean if we change the `swift-tools-version` to 5.0, we can add this `platforms` to the package file?

Reply:
OK, I found explanation for it here https://github.com/JamitLabs/Accio#adding-support-for-accio.

Reply:
Yes, exactly, the reason why I tend to rather add this Swift 4.2 version of the manifest is because not everyone has updated their projects to Swift 5 yet, I think that takes around 3 months until most project are updated. But I'm personally totally fine with already updating the manifest format to Swift 5, I just haven't made different kinds of tests with it yet when using older projects, so keeping this at 4.2 is a safer bet (since Swift 5 can perfectly handle it).